apps/arweave/src/ar_nonce_limiter_server.erl

```diff
@@ -79,25 +98,11 @@ handle_cast(Cast, State) ->
 	{noreply, State}.
 
 handle_info({event, nonce_limiter, {computed_output, Args}}, State) ->
-	{SessionKey, StepNumber, Output, _PartitionUpperBound} = Args,
-	case ar_nonce_limiter:get_session(SessionKey) of
-		not_found ->
-			?LOG_WARNING([{event, computed_output_session_not_found},
-					{session_key, ar_nonce_limiter:encode_session_key(SessionKey)},
-					{step_number, StepNumber}]),
+	case is_vdf_server() of
```

Comment:
Should we just not run the `ar_nonce_limiter_server_sup` if it's not a vdf server? That would trim out the server and all server_workers as well

Reply:
It looks like we only start the workers if we have `nonce_limiter_client_peers` defined, so it's only the `_server` that's running anyways